nuget.package/App_Readme/CHANGES.md

```diff
@@ -0,0 +1,782 @@
+# LibGit2Sharp Changes
```

Comment:
We probably don't care about including this file and README.md into our package since it's intended for internal use only anyway. But we should keep the LICENSE.md.